README.md

```diff
@@ -3,10 +3,25 @@
 
 A Charmed Operator for the OAI RAN Central Unit (CU) for K8s.
 
+## Pre-requisites
+
+A Kubernetes cluster with the Multus addon enabled.
```

Comment:
Do we want to remove the possibility of deploying DU without Multus?

Reply:
Well, they do now. Currently CU and DU use single interface for all communication. That's why I'm asking, because after this change Multus will become the only option.

Reply:
It worth to discuss with the team.